client/debian/changelog

```diff
@@ -1,7 +1,8 @@
-rust-hwlib (0.9.0~ppa8) plucky; urgency=medium
+rust-hwlib (0.9.0~ppa8) questing; urgency=medium
 
   * Add integration tests to autopkgtests
   * Update debian/copyright to include vendored dependencies licenses
+  * Package the project for Questing 
```

Comment:
The changes have not been published to the PPA we use for MIR, so I've decided to update it and publish later instead of creating another release.